src/wizards/tokenSwapDealWizard/stages/proposalStage/proposalStage.html

```diff
@@ -40,7 +40,7 @@ <h2 class="sidebarHeader heading heading3 title">Proposal</h2>
         <pform-input
           class="wizardFormInput wizardFormInputNarrow"
           label="Proposal Description"
-          label-info="Describe the details, background and features of your Token Swap proposal. Explain why it's valuable or unique."
+          label-info="Describe the details, background and features of your Token Swap proposal. Explain why it's valuable or unique"
```

Comment:
```suggestion
          label-info="Describe the details, background and features of your Token Swap proposal Explain why it's valuable or unique"
```

@dkent600 There are 2 "sentences" here, so there is still one period left after "... Swap proposal."
Are we fine with periods in the middle of the tooltip?

Reply:
yes.  In the middle is ok.